node/src/accountant/mod.rs

```diff
@@ -1887,6 +1916,18 @@ mod tests {
         assert_eq!(
             *more_money_received_params,
             vec![(now, vec![expected_receivable_1, expected_receivable_2])]
+        );
+        let commit_params = commit_params_arc.lock().unwrap();
+        assert_eq!(*commit_params, vec![()]);
```

Comment:
Just a suggestion, if you'd like we can just assert on the length of the vector, which will imply how many times the `commit()` function has been called. This empty `()` doesn't offer any value.